_posts/2017-09-08-scalafix-v0.5.md

```diff
@@ -0,0 +1,191 @@
+---
+category: blog
+by: Ólafur Páll Geirsson
+title: "Announcing Scalafix v0.5"
```

Comment:
Maybe make the title a bit more informative? Would be cool to mention more upfront the fact that Scalafix adds the ability to implement your own linting! There's a lot cool in this release that you could flaunt in the title to get people to click, read more, and try Scalafix out.

Reply:
How about "Catch bugs with scalafix v0.5"?